resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -41,7 +41,7 @@
 $userModel = null;
 $defaultSort = 1;
 if (isset($user)) {
-    $userModel = User::firstWhere('User', $user);
+    $userModel = User::with('playerBadges')->find($userID);
```

Comment:
Does this fetch all of the user's badges? Shouldn't `getUserGameProgressionAwards` just fetch the singular badge associated to the user/game combination?

Reply:
You're right - this is not optimal. Adjusted in latest so `getUserGameProgressionAwards` only targets the given game ID.